code/BuddyTrackerBasic/src/main.cpp

```diff
@@ -61,6 +61,7 @@ void onReceive(uint8_t packetSize) {
     while (LoRa.available()) {
         // read UUID
         uint64_t UUID = 0;
+        // TODO make shift and cast does not cause problems. Safest to mask
```

Comment:
Always concerned when shifting bits and casting between sizes. Make sure that this always works as expected.